examples/cost_effective_active_learning.py

```diff
@@ -0,0 +1,93 @@
+"""
+This is a modified implementation of the algorithm Cost Effective Active Learning
+(Pl. refer - https://arxiv.org/abs/1701.03551). This version not only picks up the 
+top K uncertain samples but also picks up the top N highly confident samples that
+may represent information and diversity. It is better than the original implementation
+as it does not involve tuning the confidence threshold parameter for every dataset.
```

Comment:
Do you have any references that can demonstrate that this implementation is better than the original? If no such studies were done, it is fine to leave the example implementation as you provided here, but in this case, please replace the word `better` with `different`.

Reply:
Thank you for the review. I have made necessary changes in the comments. Would you also suggest to have a separate implementation of the original CEAL algorithm? It would be quite easy to do so.

Also, I have implemented bayesian active learning by disagreement with monte carlo dropouts. The file is present in the latest commit. It would be nice if you can review that too.

Reply:
I don't think that the original implementation is necessary.

However, the Bayesian active learning with Monte Carlo dropouts is already added as an example ([https://github.com/modAL-python/modAL/blob/master/examples/deep_bayesian_active_learning.py](https://github.com/modAL-python/modAL/blob/master/examples/deep_bayesian_active_learning.py)). Can you remove your one from the PR and perhaps enhance the already existing example where you see fit? (For instance, replacing the model with LeNet model as you did would be great!)

Reply:
Thank you for the comment. I have deleted Bayesian Active Learning By Disagreement file. I think the already implemented Bayesian Active Learning with Monte Carlo Dropouts uses Max Entropy. I can modify it to use [BALD](https://arxiv.org/abs/1112.5745).